src/core/components/radio/index.tsx

```diff
@@ -153,6 +166,8 @@ const Radio = ({
 			type="radio"
 			css={(theme) => [radio(theme.radio && theme), cssOverrides]}
 			value={value}
+			aria-invalid={error}
```

Comment:
Related to the [`error` prop comment](https://github.com/guardian/source/pull/678/files#r553256066), this line is giving us a linting error:

```
error  The attribute aria-invalid is not supported by the role radio. This role is implicit on the element input  jsx-a11y/role-supports-aria-props
```